test/specs/reducers/app-state-reducer.spec.js

```diff
@@ -0,0 +1,18 @@
+import { AppStateReducer } from '../../../src/js/reducers/app-state-reducer';
+import { TOGGLE_WIDGET, OPEN_WIDGET, CLOSE_WIDGET } from '../../../src/js/actions/app-state-actions';
+
+const INITIAL_STATE = AppStateReducer(undefined, {});
```

Comment:
I'd say force `showAnimation` to be false in the initial state, otherwise we might switch the default to true and you wouldn't know your dispatched action is still switching it on.